### DISCUSSION TITLE
MOD ORGANIZATIONS*

### DISCUSSION THREAD

**Discussion by Equinox-76** [PrismLauncher/PrismLauncher#2344] [Ideas]
Hi, It would be really useful if you could **add a system for organizing mods into different folders**. I know many people have a large amount of mods (including myself), and organizing them would be helpful. For example, there could be a little plus button on the side tab in the mods section that adds a folder (which you can rename). Then you can drag and drop mods into or out of them.

Adding organization support for resource packs, shader packs, notes, worlds, servers, and screenshots would be nice too.

**Comment by roke-julian-lockhart** [comment]
> https://github.com/orgs/PrismLauncher/discussions/2344#discussion-6559101

@Equinox-76, your title doesn't describe this suggestion well. You should name it something like "Mod folders to organize mods".
